engine/src/test/scala/cromwell/services/CromwellServicesTest.scala

```diff
@@ -0,0 +1,45 @@
+package cromwell.services
+
+import java.util.UUID
+
+import akka.actor.{ActorRef, ActorSystem}
+import com.typesafe.config.{Config, ConfigFactory}
+import cromwell.backend.{BackendJobDescriptor, BackendJobDescriptorKey, BackendWorkflowDescriptor}
+import cromwell.core.{WorkflowId, WorkflowOptions}
+import cromwell.engine.WorkflowSourceFiles
+import cromwell.engine.backend.{WorkflowDescriptor, WorkflowDescriptorBuilder}
+import cromwell.engine.db.DataAccess
+import spray.json.JsObject
+
+trait CromwellServicesTest extends WorkflowDescriptorBuilder {
+  val actorSystem: ActorSystem
+
+  protected def getBackendJobDescriptorKey(descriptor: WorkflowDescriptor, callName: String): BackendJobDescriptor = {
+    val call = descriptor.namespace.workflow.calls.find(_.unqualifiedName == callName).get
```

Comment:
_This is ~~PTSD~~ (Current)TSD dealing with merging FinalCallKey to Backend/CallKey._

How about a more descriptive error, `.getOrElse(throw new Exception("you messed up…"))`.